frontend/src/components/auth/components/sign-up-form/sign-up-form.module.scss

```
@@ -0,0 +1,42 @@
+.signUpTitle,
+.signUpForm,
+.signInLink,
+.signUpFormPolicy,
+.signUpLogo {
+  width: 25%;
+  margin: 0 auto;
+  color: #a0a0a0;
+  font-family: Montserrat, sans-serif;
```

```suggestion
  font-family: Montserrat, sans-serif;
```
just add it as a global